contracts/governance/BaseWeightedMultisig.sol

```diff
@@ -207,7 +238,7 @@ abstract contract BaseWeightedMultisig is IBaseWeightedMultisig {
      * @notice Gets the specific storage location for preventing upgrade collisions
      * @return slot containing the WeightedMultisigStorage struct
      */
-    function _baseWeightedStorage() private pure returns (WeightedMultisigStorage storage slot) {
+    function _storage() private pure returns (WeightedMultisigStorage storage slot) {
```

Comment:
I think base contract should have named storage function. We better use `_storage()` in derived contracts: Multisig and Auth. Lets revert it here

Reply:
solidity issue for it is still open it seems
https://github.com/ethereum/solidity/pull/14357